src/integrate/totalvi/script.py

```diff
@@ -0,0 +1,181 @@
+from typing import Tuple
+
+import logging
+import mudata
+from anndata import AnnData  # For type hints
+from mudata import MuData  # For type hints
+import numpy as np
+from pandas import DataFrame
+import scvi
+from scipy.sparse import issparse
+from torch.cuda import is_available as cuda_is_available
+try:
+    from torch.backends.mps import is_available as mps_is_available
+except ModuleNotFoundError:
+    # Older pytorch versions
+    # MacOS GPUs
+    def mps_is_available():
+        return False
+    
+import os
+
+### VIASH START
+par = {
+    "query": "resources_test/scvi_tools/totalvi_pbmc_query_test.h5mu",
+    "reference": "resources_test/scvi_tools/totalvi_pbmc_ref_test.h5mu",
+    "query_modality": "rna",
+    "query_proteins_key": None,
+    "reference_modality": "rna",
+    "reference_proteins_key": None,
+    "force_retrain": False,
+    "input_layer": None,
+    "obs_batch": "sample_id",
+    "var_input": None,
+    "output": "foo.h5mu",
+    "obsm_output": "X_integrated_totalvi",
+    "reference_model_path": "totalvi_model_reference/",
+    "query_model_path": "totalvi_model_query/",
+    "max_epochs": 400,
+    "max_query_epochs": 200,
+    "weight_decay": 0.0
+}
+### VIASH END
+
+
+def _setup_logger():  # Is there a place where to put common code to not repeat it across components?
```

Comment:
Good question. There are two solutions for this:
(1) We could create a folder "common" or something like that and put scripts in there that contain the common code. These scripts can be included in the the component using the "resources" and imported.
(2) create a python package and use pip install

This is a good suggestions but I am not considering this blocking for this PR.

Reply:
I would fix this in a separate PR

Reply:
Indeed, lets not make this blocking for this PR